File: app/src/main/java/com/renderson/currency_converter/MyApplication.java
package com.renderson.currency_converter;

import android.app.Application;
import android.content.Context;

import androidx.appcompat.app.AppCompatDelegate;

import dagger.hilt.android.HiltAndroidApp;

@HiltAndroidApp
public class MyApplication extends Application {
    private static MyApplication instance;

    public void onCreate() {
        instance = this;
        super.onCreate();
        AppCompatDelegate.setDefaultNightMode(
                AppCompatDelegate.MODE_NIGHT_NO);
    }

    public static Context getContext() {
        return instance;
    }
}

File: app/src/main/java/br/com/conversordemoedas/MyApplication.kt
package br.com.conversordemoedas

import android.app.Application
import br.com.conversordemoedas.di.DependencyModules
import org.koin.android.ext.android.startKoin

class MyApplication: Application() {

    override fun onCreate() {
        super.onCreate()

        startKoin(this, listOf(DependencyModules.appModule))

    }

}